modules/libcom/src/error/errSymLib.c

```diff
@@ -23,25 +23,25 @@
 #include "cantProceed.h"
 #include "epicsAssert.h"
 #include "epicsStdio.h"
+#include "epicsAtomic.h"
 #include "errMdef.h"
 #include "errSymTbl.h"
 #include "ellLib.h"
 #include "errlog.h"
 
+#define MIN_MODULE_NUM 501
 #define NHASH 256
 
 static epicsUInt16 errhash(long errNum);
 
 typedef struct errnumnode {
-    ELLNODE            node;
     long               errNum;
     struct errnumnode *hashnode;
     const char        *message;
     long               pad;
 } ERRNUMNODE;
 
-static ELLLIST errnumlist = ELLLIST_INIT;
-static ERRNUMNODE **hashtable;
+static ERRNUMNODE *hashtable[NHASH * sizeof(ERRNUMNODE *)];
```

Comment:
Inappropriate use of `sizeof`: this a static array, not `malloc()`, so its size is given as number of elements, not bytes.

Reply:
Please note that we are allocating pointers, so, what we have is the same as `sizeof(void *)`

Reply:
Let me put this another way. The following three statements allocate an object of the same size:
```
void *dynmem = malloc(num_elements * sizeof(void*));
void *zeromem = calloc(num_elements, sizeof(void*));
void *staticmem[num_elements];
```

Reply:
Ohh, now I understand, thanks for spotting that, I'll fix it